app/common/templates/common/auth/claim_magic_link.html

```diff
@@ -16,3 +16,7 @@
         </div>
     </div>
 {% endblock content %}
+
+{% block bodyEnd %}
+  <script nonce="{{ csp_nonce() }}">document.getElementById('submit').click();</script>
```

Comment:
I'll give this a go in a bit to get a sense for the page loading/ user experience but if this visually loads the page before the JavaScript runs I think we could try putting the js snippet in the `<head>` tag and having it either update the `window.locationl.href` or possibly create the form/ button and click it (to keep it POST-y) before the page has been fully fetched/ rendered.

Reply:
https://github.com/alphagov/notifications-admin/blob/e5a228cc8e4b49f2e2b32162d6b34e22bb6d347b/app/templates/views/email-link-interstitial.html

similar

Reply:
https://github.com/communitiesuk/funding-service/pull/46/commits/5b257f8f83b2ca89961d859bbae45107608cc112